src/scripts/lib/data.js

```diff
@@ -0,0 +1,1105 @@
+export const data = [
```

Comment:
@hasnaindev The data is still stored in a `.js` file, could we not use a `.json` file and import it where required?

Reply:
Sure, I will change this to JSON and make another pull request.